README.md

```diff
@@ -116,7 +116,7 @@ bcrypt.genSalt(saltRounds, function(err, salt) {
 Technique 2 (auto-gen a salt and hash):
 
 ```javascript
-bcrypt.hash(myPlaintextPassword, saltRounds, function(err, hash) {
+bcrypt.hash(myPlaintextPassword, salt, function(err, hash) {
```

Comment:
If you supply the parameter as an integer, it will automatically call genSalt for you